src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -327,15 +387,16 @@ private ListBoxModel getEnvironmentIdItems(Secret formApiKey, boolean disableSsl
             return items;
         }
 
-        public ListBoxModel doFillLabelsItems(@QueryParameter Secret restApiKey, @QueryParameter boolean disableSslVerification) {
-            if (isSecretEmpty(restApiKey)) {
+        public ListBoxModel doFillLabelsItems(@QueryParameter String restApiKeyName, @QueryParameter boolean disableSslVerification) {
+            if (StringUtils.isBlank(restApiKeyName)) {
                 ListBoxModel items = new ListBoxModel();
                 items.add("<No Labels Found>", "");
```

Comment:
Is this message correct? No labels found?

Reply:
Good catch. I should change this to show "Input an API key" instead.